db/migrate/20230125035957_create_patients.rb

```diff
@@ -0,0 +1,16 @@
+class CreatePatients < ActiveRecord::Migration[7.0]
+  def change
+    create_table :patients do |t|
+      t.string :name
+      t.string :phone
+      t.string :cpf
+      t.string :email
+      t.string :birthdate
+
+      t.timestamps
+    end
+
+    add_index :patients, :email, unique: true
```

Comment:
nao sei se é interessante colocar o email como unico

Reply:
Why not?